File: benchmarks/string/string.py
import sys
import time

N = int(sys.argv[1])

t0 = time.perf_counter()

a = "a"
b = "b"
while N > 0:
  a = a + b
  b = b + a
  N -= 1


t1 = time.perf_counter()

print("Length of a: " + str(len(a)))
print("Execution took (ms) " + str((t1 - t0) * 1000))

File: benchmarks/string/string.lua

local t0 = os.clock()
local N = tonumber(arg[1])

local a = "a"
local b = "b"
while N > 0 do
    a = a .. b
    b = b .. a
    N = N - 1
end

local t1 = os.clock()

print("Length of a: " .. string.len(a))

print("Execution took (ms) " .. ((t1 - t0) * 1000))